response-to-reviewers.md

```diff
@@ -285,11 +287,16 @@ See also our response to related comments made by Reviewers #1 and #2.
 
 > 2) I like the inclusion of the "zigzag" history, as well as generic piecewise constant models and IM models (page 7). I wonder if these could be included in a separate section (not organism specific) in the documentation and software (and then in Table 1). Right now the zigzag model is under humans in the catalog.
 
-This is an interesting idea. We have added suite of generic models to stdpopsim that do not necessarily capture demographic parameters from a certain species,
-but do reflect general trends observed in inference. The "zigzag" model could definitely be one of these, and indeed was initially implemented as a generic. However the reason we settled on the zigzag model is being defined as a human model is that its
-effective population size values are taken from (or at least inspired by) values inferred from human genomes. To see our internal discussion of these issues from September 2019 please see this [closed issue](https://github.com/popsim-consortium/stdpopsim/issues/106).
+We considered both of these suggestions, and here's why we've left it the way it is.
+Most of the columns of the table don't apply to generic models,
+so it seems strange to include them there.
+The "zigzag" model could definitely be a generic model,
+and indeed was initially implemented as such.
+However, the reason we settled on the zigzag model is being defined as a human model is that its
+effective population size values are taken from (or at least inspired by) values inferred from human genomes: as implemented, it is not as "generic" as it might seem.
+
```

Comment:
The issue didn't seem all that helpful, really.

Reply:
happy to delete it, but man it's sweet to be able to point to github and say "look reviewers we talked about this!"

Reply:
good point; put this back in